src/components/search/index.tsx

```diff
@@ -0,0 +1,96 @@
+import Downshift from 'downshift';
+import { useState } from 'react';
+import { useAutocomplete } from '@atb/page-modules/departures/client';
+import style from './search.module.css';
+import { split } from 'lodash';
+
+type SearchProps = {
+  label: string;
+  onChange: (selection: any) => void;
+};
+
+export default function Search({ label, onChange }: SearchProps) {
+  const [query, setQuery] = useState('');
+  const { data } = useAutocomplete(query);
```

Comment:
This was just a simple example. I think for this to be used in a text input like this we should introduce some sort of debouncing to reduce the number of requests 🤔

Reply:
(inside the `useAutocomplete` hook)

Reply:
Agree, was just thinking about that. Adding it to the TODO list.